<details><summary>Failed to submit graph for e62ebc0d75329df704e91e82c422886a324d2a29
</summary>Message:
Failed to run command: 'pbp graph create --host https://package-build.internal.anaconda.com/pkg-orchestrator -f /tmp/tmp28nelbng/clang-compiler-activation-feedstock.yaml --with-linter', with error: 'INFO:root:Loading Graph Data from /tmp/tmp28nelbng/clang-compiler-activation-feedstock.yaml
INFO:pbpcli:Creating Graph with group name: clang-compiler-activation-feedstock/pull/25/commits/e62ebc0
INFO:root:b'{"detail":[{"type":"value_error","loc":["body","tasks",0,"build","build_options"],"msg":"Value error, Invalid build_options: [\'-c https://staging.continuum.io/prefect/llvm-20.1.8-stage2-build-0\']. Allowed: [\'--debug\', \'--error-overdepending\', \'--error-overlinking\', \'--no-error-overlinking\', \'--no-include-recipe\', \'--no-test\', \'--no-verify\', \'--quiet\', \'--skip-existing\', \'--suppress-variables\']","input":["--suppress-variables","-c https://staging.continuum.io/prefect/llvm-20.1.8-stage2-build-0"],"ctx":{"error":{}}}]}'
Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/bin/pbp", line 10, in <module>
    sys.exit(cli())
             ~~~^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1442, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1363, in main
    rv = self.invoke(ctx)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1226, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 794, in invoke
    return callback(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/graph.py", line 187, in create
    response = create_graph(
        host, graph, with_linter=with_linter, release=release, autopublish=autopublish
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/utils/graph_utils.py", line 197, in create_graph
    raise Exception("Failed to create graph")
Exception: Failed to create graph
'
aws lambda request id: 8650f804-b1d4-5865-9992-9a262d5f8bf7
[View in Grafana](https://grafana.anacondaconnect.com/explore?schemaVersion=1&panes=%7B%2286g%22:%7B%22datasource%22:%22loki-central%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7B__aws_cloudwatch_log_group%3D%5C%22%2Faws%2Flambda%2Fpbp-graph-submitter-prod%5C%22%7D%20%7C%20json%20%7C%20requestId%20%3D%20%608650f804-b1d4-5865-9992-9a262d5f8bf7%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki-central%22%7D,%22editorMode%22:%22builder%22,%22direction%22:%22backward%22%7D%5D,%22range%22:%7B%22from%22:%22now-3h%22,%22to%22:%22now%22%7D,%22panelsState%22:%7B%22logs%22:%7B%22columns%22:%7B%220%22:%22Time%22,%221%22:%22Line%22%7D,%22visualisationType%22:%22table%22,%22labelFieldName%22:%22labels%22%7D%7D%7D%7D&orgId=1)</details>

--- a/abs.yaml
+++ b/abs.yaml
@@ -1,5 +1,4 @@
 # the conda build parameters to use
 build_parameters:
   - "--suppress-variables"
-  - "-c https://staging.continuum.io/prefect/llvm-20.1.8-stage2-build-0"
 

